Entities/Actor.h

```diff
@@ -914,16 +914,12 @@ ClassInfoGetters
 
 	const std::deque<MovableObject *> * GetInventory() { return &m_Inventory; }
 
+	/// <summary>
+	/// Returns the maximum total mass this Actor can carry in its inventory.
+	/// </summary>
+	/// <returns>The maximum carriable mass of this Actor.</returns>
 
-//////////////////////////////////////////////////////////////////////////////////////////
-// Method:  GetMaxMass
-//////////////////////////////////////////////////////////////////////////////////////////
-// Description:     Tells how much total mass (in kg) this Actor is recommended to weigh
-//                  at most INCLUDING his own weight AND all his inventory!
-// Arguments:       None.
-// Return value:    The max recommend total mass for this Actor
-
-	float GetMaxMass() const { return m_MaxMass; }
+	float GetMaxInventoryMass() const { return m_MaxInventoryMass; }
```

Comment:
Remove whitespace between declaration and comment when reformatting otherwise VS doesn't associate them and hovering over a method call doesn't show the description.
Same thing happens with the old comment style be we don't care so no need to fix those.